omero/developers/Web/CreateApp.txt

```diff
@@ -67,49 +69,70 @@ OR you could simply choose an existing location:
 Create and check out a new GitHub repository OR manually create a new directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
--  Login to your GitHub account homepage (e.g.
-   `<https://github.com/>`_\ <your-name>/) and click
+-  Login to your GitHub account homepage e.g.
+   `<https://github.com/>`_\<your-name>/) and click
    "New repository"
--  Enter the name of <your-app>, add a description and choose to add a
-   README.
 
--  Check out your new repository (into a new directory)
+-  Enter the name of your repository i.e. ``<organisation-app>`` e.g. ``omero-webtest``,
+   add a description and choose to add a ``README.rst``. Change the extension to ``.rst`` so
+   it can be rendered correctly on PyPI. Its contain should cover the goal of the app
+   and the configuration instructions.
+
+-  Check out your new repository into a new directory
 
    ::
 
-       $ git clone git@github.com:<your-name>/<your-app>.git
+       $ git clone git@github.com:<your-name>/<organisation-app>.git
 
 
 -  OR: If you have not used git to create your app directory above, then
 
    ::
 
-        $ mkdir <your-app>
+        $ mkdir <organisation-app>
 
--  In either case, you should now have a directory called ``your-app`` within
+-  In either case, you should now have a directory called ``organisation-app`` within
    a directory that is on your :envvar:`PYTHONPATH`.
 
+Make your app installable from PyPI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is not required but it is highly recommended to make your app
+installable from PyPI. Few files need to be added:
+
+- A ``MANIFEST.in`` file. This is needed in certain cases to package files not automatically included
```

Comment:
I think these would read better if you formatted them as
"``filename.ext`` - a [file type] that is..."